Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealitySpeechCommandsProfile.asset

```diff
@@ -27,22 +28,9 @@ MonoBehaviour:
       id: 1
       description: Toggle Diagnostics
       axisConstraint: 0
-  - keyword: Toggle Frame Rate
-    keyCode: 102
-    action:
-      id: 2
-      description: Toggle Frame Rate
-      axisConstraint: 0
-  - keyword: Toggle Processor
+  - keyword: Toggle Profiler
     keyCode: 112
     action:
-      id: 3
-      description: Toggle Processsor
-      axisConstraint: 0
-  - keyword: Toggle Memory
-    keyCode: 109
-    action:
-      id: 4
-      description: Toggle Memory
-      axisConstraint: 0
-
+      id: 7
+      description: Grip Press
```

Comment:
Is Grip Press intentional here?

Reply:
the older version had grip press if i recall correctly,

Reply:
just confirmed, no custom actions and grip pressed was used.